starburst.gemspec

```diff
@@ -16,9 +16,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.rdoc"]
 
-  s.add_dependency "rails", ">= 4.2.0", "< 6.0"
-
-  s.add_development_dependency "sqlite3", "~> 1.3.9"
```

Comment:
We need to keep this here to support local development without `Appraisal`. Moving the `sqlite3` line together with the other development dependencies would be good though.